app/Models/Post.php

```diff
@@ -80,13 +80,41 @@ public function reactions()
     public function getMediaUrl()
     {
         if ($this->url === 'default') {
+            // @TODO: We should either return a placeholder, or `null`.
             return 'default';
         }
 
+        // If Glide is enabled, provide the default image URL.
+        if (config('features.glide')) {
+            return url('images/' . $this->id);
+        }
+
         // Ask the storage driver for the path to the image for this post.
-        // @TODO: Update this to provide a default Glide URL!
         $path = Storage::url('uploads/reportback-items/edited_' . $this->id . '.jpeg');
 
         return url($path);
     }
+
+    /**
+     * Return the filesystem path for this post.
+     *
+     * @return string|null
+     */
+    public function getMediaPath()
+    {
+        // We store local URLs prefixed with `/storage` when using the "public" adapter.
+        if (config('filesystems.default') == 'public') {
+            return str_replace('/storage/', '', $this->url);
```

Comment:
Yep, it's a little wonky! Since `/storage` is the "base path" for the storage adapter, we need to remove it when asking for a file path (but add it when giving a URL to the user).